io/src/main/scala/sbt/io/Milli.scala

```diff
@@ -0,0 +1,364 @@
+package sbt.io
```

Comment:
I think this file should move to `sbt.internal.io` package, and we should provide something like `IO.modifiedTime(file)`, etc in `IO`.

Also please copy what you wrote in the pull request description into this commit's message, so we can read it later.

Reply:
Also some unit test might be helpful (even though we'll mostly run only Linux one via Travis).

Reply:
Sure, it's already planned. Unit testing is coming up as soon as the main validation passes.